### PR TITLE
[PR] Layout of window's icon

### LINKED ISSUE #691: [FR] Layout of window's icon
The layout of the window's icon is fine in expand mode. But it is overlaid with the document title when you're in the smallest window's size.

image

Smallest window's size mode.
image

### PR DESCRIPTION
**Pull request by victor-teles**
fixes #691

### CODE REVIEW

File: frontend/app_flowy/lib/workspace/presentation/home/home_layout.dart [hunk 1]
```diff
@@ -37,6 +38,7 @@ class HomeLayout {
     }
 
     homePageLOffset = showMenu ? menuWidth : 0.0;
+    menuSpacing = showMenu ? 0 : 80.0;
```
Comment (vincentdchan) [inline frontend/app_flowy/lib/workspace/presentation/home/home_layout.dart:41]: I suppose this space only shows on macOS. Maybe you need `Platform.isMacOS`.

Comment (victor-teles) [reply frontend/app_flowy/lib/workspace/presentation/home/home_layout.dart]: @vincentdchan done!

**Comment by vincentdchan** [comment]
CI doesn't pass.

**Comment by victor-teles** [comment]
@vincentdchan i think it's ok now

**Comment by vincentdchan** [comment]
Thanks for your contributions!

**Pull request merged**